### ISSUE TITLE
10$: suggest where to write 'async' if missing

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#1037]
The following code reports E0162:
```javascript
import fs from "node:fs";

function loadJSONFile(path) {
  let data = await fs.promises.readFile(path, "utf-8");
  //         ^^^^^ E0162: 'await' is only allowed in async functions
}
```
The diagnostic should have a second message pointing before the `function` keyword. This is where the user would type 'async'. We would eventually use this for auto fixing (#115).
